pkg/oc/originpolymorphichelpers/logsforobject.go

```diff
@@ -39,6 +45,29 @@ func NewLogsForObjectFn(delegate polymorphichelpers.LogsForObjectFunc) polymorph
 				return nil, err
 			}
 			return appsmanualclient.NewRolloutLogClient(appsClient.Apps().RESTClient(), t.Namespace).Logs(t.Name, *dopts), nil
+		case *appsapiv1.DeploymentConfig:
+			dopts, ok := options.(*appsapiv1.DeploymentLogOptions)
+			if !ok {
+				return nil, errors.New("provided options object is not a DeploymentLogOptions")
+			}
+			appsClient, err := appsclientv1.NewForConfig(clientConfig)
+			if err != nil {
+				return nil, err
+			}
+			return appsmanualclientv1.NewRolloutLogClient(appsClient.RESTClient(), t.Namespace).Logs(t.Name, *dopts), nil
+		case *buildapiv1.Build:
+			bopts, ok := options.(*buildapiv1.BuildLogOptions)
+			if !ok {
+				return nil, errors.New("provided options object is not a v1.BuildLogOptions")
+			}
+			if bopts.Version != nil {
+				return nil, errors.New("cannot specify a version and a build")
+			}
+			buildClient, err := buildclientv1.NewForConfig(clientConfig)
+			if err != nil {
+				return nil, err
+			}
+			return buildmanualclientv1.NewBuildLogClient(buildClient.RESTClient(), t.Namespace).Logs(t.Name, *bopts), nil
 		case *buildapi.Build:
```

Comment:
Additionally, since we've just migrated logs to external, do we need internals here?